### ISSUE TITLE
torrent-file-editor-0.3.18 Found unsuitable Qt version "5.15.2" 

### ISSUE THREAD

**Issue by slrslr** [torrent-file-editor/torrent-file-editor#154]
I have downloaded and extracted https://github.com/torrent-file-editor/torrent-file-editor/releases/download/v0.3.18/torrent-file-editor-0.3.18.tar.gz
went to the extracted directory, ran (per README.md file):
```
    mkdir build && cd build
    cmake -DCMAKE_BUILD_TYPE=Release -DQT5_BUILD=OFF ..
    make
```

it failed at second command:
```
CMake Error at /usr/share/cmake-3.18/Modules/FindQt4.cmake:1314 (message):
  Found unsuitable Qt version "5.15.2" from /usr/bin/qmake, this code
  requires Qt 4.x
Call Stack (most recent call first):
  CMakeLists.txt:160 (find_package)


-- Configuring incomplete, errors occurred!
See also "/home/user/Downloads/torrent-file-editor-0.3.18/build/CMakeFiles/CMakeOutput.log".

$ cat "/home/user/Downloads/torrent-file-editor-0.3.18/build/CMakeFiles/CMakeOutput.log"
$ (empty)
```

though the github page https://github.com/torrent-file-editor/torrent-file-editor/releases/tag/v0.3.18 shows "Add Qt6 support" yet as per the output above, it requires qt 4 and not accepts qt 5? Can you suggest any workaround? I am on Debian 11 (oldstable).

**Comment by drizt** [comment]
`-DQT5_BUILD=OFF` - wrong. It's not used now. Use 
```
mkdir build && cd build
cmake -DCMAKE_BUILD_TYPE=Release -DQT4_BUILD=ON ..
make
```

for your case.

Anyway seems problem in your qt4/qt5 packages. I tried to implement correct Qt version detecting. Show `qmake -query` output. Also show `ls /usr/bin/qmake*`.

**Comment by slrslr** [comment]
@drizt [Here](https://paste.nolog.cz/?741e3066cfdbd216#3pgRkfgCLM4FE8FYtPoFe6mzCtCHXEu7NXSVHEW7mQEM) I have posted the output of your commands.

**Comment by drizt** [comment]
You need to compile with Qt5. Not Qt4. Your system provides Qt5. Use `-DQT5_BUILD=ON` or skip this parameter. Probably no Qt4 your system. Check Qt packages.

**Comment by slrslr** [comment]
Thank you, this worked:
`cmake -DCMAKE_BUILD_TYPE=Release -DQT5_BUILD=ON ..`
Yet produced one more error:
```
CMake Error at CMakeLists.txt:142 (find_package):
  By not providing "FindQt5LinguistTools.cmake" in CMAKE_MODULE_PATH this
  project has asked CMake to find a package configuration file provided by
  "Qt5LinguistTools", but CMake did not find one.
```
which was solved by running "[sudo apt install qttools5-dev](https://stackoverflow.com/a/51698165)" and repeating the cmake command and then "make".

If you think that the README.md mentioned parameter "-DQT5_BUILD=OFF" is correct (for me not worked, but worked "-DQT5_BUILD=ON"), i guess this issue can be closed. Thank you

**Comment by drizt** [comment]
Probably you looked at README.md from tarball. Please look at README on github master branch. It was corrected after 0.3.18 version.

**Comment by slrslr** [comment]
indeed, it is correct in README here on github
